nservicebus/nhibernate/saga-concurrency.md

```diff
@@ -30,6 +30,8 @@ That property will be included by NHibernate in the `SELECT` and `UPDATE` SQL st
 
 NOTE: Marking a property with `RowVersion` **does not** disable the pessimistic locking optimization. All it does is replacing the default optimistic concurrency validation that depends on values of all columns with one that is based on that single explicit version column. To switch to pure optimistic concurrency adjust the locking strategy to `Read`.
 
+NOTE: NHibernate doesn't support `RowVersion` attribute when used on derived classes. Instead of inheriting saga data from the `ContainSagaData` class, directly implement the `IContainSagaData` interface.
```

Comment:
@Particular/nhibernate-persistence-maintainers  - should we make this as a warning more prominently?